Nero/plugins/Nero.cc

```diff
@@ -190,7 +222,7 @@ Nero::Nero(const edm::ParameterSet& iConfig)
     phos -> mMinPt = iConfig.getParameter<double>("minPhoPt");
     phos -> mMaxIso = iConfig.getParameter<double>("maxPhoIso");
     phos -> mMinNpho = iConfig.getParameter<int>("minPhoN");
-    phos -> mMaxEta = iConfig.getParameter<double>("maxPhoEta");
+    phos -> mMaxEta = iConfig.getParameter<double>("minPhoEta");
```

Comment:
Ah sorry, I was trying to make it uniform. I went with the one that's more common :)